flow-typed/content.js

```diff
@@ -2,15 +2,16 @@
 
 declare type ContentState = {
   primaryUri: ?string,
-  playingUri: { uri?: string },
+  playingUri: {
+    uri?: string,
+    collection: PlayingCollection,
+  },
   positions: { [string]: { [string]: number } }, // claimId: { outpoint: position }
   history: Array<WatchHistory>,
   recommendationId: { [string]: string }, // claimId: recommendationId
   recommendationParentId: { [string]: string }, // claimId: referrerId
   recommendationUrls: { [string]: Array<string> }, // claimId: [lbryUrls...]
   recommendationClicks: { [string]: Array<number> }, // "claimId": [clicked indices...]
-  loopList?: { collectionId: string, loop: boolean },
-  shuffleList?: { collectionId: string, newUrls: Array<string> | boolean },
   // TODO: it's confusing for newUrls to be a boolean --------- ^^^
   // It can/should be '?Array<string>` instead -- set it to null, then clients
   // can cast it to a boolean. That, or rename the variable to `shuffle` if you
```

Comment:
This comment can be removed since the code is gone, plus the new implementation down below is using `?{ }` :+1: